Kitodo/src/test/java/org/kitodo/production/services/data/ImportServiceIT.java

```diff
@@ -199,6 +201,45 @@ public void testImportProcessWithAdditionalMetadata() throws DAOException, Impor
         }
     }
 
+    /**
+     * Tests whether basic catalog metadata import with additional preset metadata including LABEL and ORDERLABEL
+     * succeeds.
+     *
+     * @throws DAOException when loading ImportConfiguration or removing test process from test database fails.
+     * @throws ImportException when importing metadata fails
+     * @throws IOException when importing metadata fails
+     */
+    @Test
+    public void testImportProcessWithAdditionalMetadataWithLabelAndOrderlabel() throws DAOException, ImportException, IOException {
+        Map<String, List<String>> presetMetadata = new HashMap<>();
+        presetMetadata.put(TITLE, List.of("Band 1"));
+        presetMetadata.put(PLACE, List.of("Hamburg", "Berlin"));
+        presetMetadata.put(LABEL, List.of("TEST-LABEL"));
+        presetMetadata.put(ORDERLABEL, List.of("TEST-ORDERLABEL"));
+        Process processWithAdditionalMetadata = importProcessWithAdditionalMetadata(RECORD_ID,
+                MockDatabase.getK10PlusImportConfiguration(), presetMetadata);
+        Workpiece workpiece = ServiceManager.getMetsService()
+                .loadWorkpiece(processService.getMetadataFileUri(processWithAdditionalMetadata));
+        HashSet<Metadata> metadata = workpiece.getLogicalStructure().getMetadata();
+        String processLabel = workpiece.getLogicalStructure().getLabel();
+        String processOrderlabel = workpiece.getLogicalStructure().getOrderlabel();
+        try {
+            Assert.assertTrue("Process does not contain correct metadata",
```

Comment:
After PR #6093 is merged the old Junit4 based tests must be adjusted to be Junit5 based tests. In most cases the used parameters must be switched and  `Assert` must be replaced with `Assertions` or better don't use them directly in code and use them as static import.